src/ansys/grantami/bomanalytics/_item_definitions.py

```diff
@@ -396,38 +482,79 @@ def _definition(self) -> models.GetComplianceForSubstancesSubstanceWithAmount:
         return definition
 
 
-class CoatingReference(RecordReference):
-    """Extends RecordReference without changes, to re-define the class name, because it appears in the repr."""
+class CoatingReference(IdentifierMixin, RecordReference):
+    """Represents a reference to a coating record."""
 
+    def __init__(
+        self,
+        equivalent_references: Optional[list["CoatingReference"]] = None,
+        **kwargs: Any,
+    ) -> None:
+        super().__init__(**kwargs)
+        self._equivalent_references = equivalent_references
 
-class CoatingReferenceWithIdentifier(IdentifierMixin, CoatingReference):
-    pass
+    @property
+    def equivalent_references(self) -> Optional[list["CoatingReference"]]:
+        """
+        Other coating records which are defined as being equivalent to this record.
+
+        Only populated if this record was included in the analysis via a link from a record in an external database.
+
+        .. versionadded:: 2.4
+        """
+        return self._equivalent_references
```

Comment:
This line is not executed in tests, and I can't think how to do it. Coatings are never provided explicitly as the input to a request, so they will never be a thing with equivalents. I've asked Aleks to see if I'm missing something here.

Reply:
I have confirmed that currently coatings cannot be specified as an input, so a response will never be returned from the server with this populated.